docs/setup/settings.asciidoc

```diff
@@ -296,14 +308,137 @@ This value must be a whole number greater than zero. *Default: `"1000"`*
+the `polling` method could be used enabling that option. *Default: `false`*
+
+`logging.silent`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+ Instead, set the root logger `level` to `off`.
+ +
+Set the value of this setting to `true` to
+suppress all logging output. *Default: `false`*
+
+`logging.timezone`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, use the {kibana-ref}/logging-service.html#pattern-layout[`pattern` appender layout] and specify a `%date` conversion pattern.
+ +
+Set to the canonical time zone ID
+(for example, `America/Los_Angeles`) to log events using that time zone.
+For possible values, refer to
+https://en.wikipedia.org/wiki/List_of_tz_database_time_zones[database time zones]. *Default: `UTC`*
+
+[[logging-verbose]] `logging.verbose` {ess-icon}::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, set the root logger `level` to `debug`.
+ +
+Set to `true` to log all events, including system usage information and all
+requests. *Default: `false`*
+
 NOTE: To reload the <<logging-settings, logging settings>>, send a SIGHUP signal to {kib}.
 
+// new logging configuration settings
 [[logging-root]] `logging.root`::
-The `root` logger has is a <<dedicated-loggers, dedicated logger>> and is pre-configured. The `root` logger logs at `info` level by default. If any other logging configuration is specified, `root` _must_ also be explicitly configured.
+The `root` logger has is a <<dedicated-loggers, dedicated logger>> and is pre-configured. The `root` logger logs at `info` level by default. If any other logger is configured, the `root` logger _must_ also be explicitly configured with an appropriate appender. 
 
 [[logging-root-level]] `logging.root.level` {ess-icon}::
-Level at which a log record should be logged. Supported levels are: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_. Levels are ordered from _all_ (highest) to _off_ and a log record will be logged it its level is higher than or equal to the level of its logger, otherwise the log record is ignored. Use this value to <<change-overall-log-level,change the overall log level>>. Set to `all` to log all events, including system usage information and all requests. Set to `off` to silence all logs. *Default: `info`*.
+Level at which a log record should be logged. Supported levels are: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_. Levels are ordered from _all_ (highest) to _off_ and a log record will be logged it its level is higher than or equal to the level of its logger, otherwise the log record is ignored. Use this value to <<change-overall-log-level,change the overall log level>>. Set to `all` to log all events, including system usage information and all requests. Set to `off` to silence all logs. *Default: `info`*. 
```

Comment:
grammar nit:
```suggestion
Level at which a log record should be logged. Supported levels are: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_. Levels are ordered from _all_ (highest) to _off_, and a log record will be logged it its level is higher than or equal to the level of its logger, otherwise the log record is ignored. Use this value to <<change-overall-log-level,change the overall log level>>. Set to `all` to log all events, including system usage information and all requests. Set to `off` to silence all logs. *Default: `info`*. 
```

Reply:
The Oxford comma gets me every time!